airbyte-integrations/connectors/source-jdbc/src/testFixtures/java/io/airbyte/integrations/source/jdbc/test/JdbcSourceStandardTest.java

```diff
@@ -145,31 +163,58 @@ public void setup() throws Exception {
     if (supportsSchemas()) {
       createSchemas();
     }
+
+    if (getDriverClass().toLowerCase().contains("oracle")) {
+      database.execute(connection -> connection.createStatement()
+          .execute("ALTER SESSION SET NLS_DATE_FORMAT = 'YYYY-MM-DD'"));
+    }
+
     database.execute(connection -> {
-      connection.createStatement()
-          .execute(String.format("CREATE TABLE %s(id INTEGER, name VARCHAR(200), updated_at DATE, PRIMARY KEY (id));",
-              getFullyQualifiedTableName(TABLE_NAME)));
+
       connection.createStatement().execute(
           String.format(
-              "INSERT INTO %s(id, name, updated_at) VALUES (1,'picard', '2004-10-19'),  (2, 'crusher', '2005-10-19'), (3, 'vash', '2006-10-19');",
+              "CREATE TABLE %s(id INTEGER, name VARCHAR(200), updated_at DATE, PRIMARY KEY (id))",
+              getFullyQualifiedTableName(TABLE_NAME)));
+      connection.createStatement().execute(
+          String.format("INSERT INTO %s(id, name, updated_at) VALUES (1,'picard', '2004-10-19')",
```

Comment:
does oracle db not support multi line inserts? we can't just remove the semi colons here? we need to split it into 4 separate inserts?

Reply:
Oracle doesn't support multi-insert. Unfortunately, we need to break into individual inserts and remove semi-colons to be compatible Oracle with others databases.
https://stackoverflow.com/questions/39576/best-way-to-do-multi-row-insert-in-oracle
Another example from [Oracle tutorial](https://livesql.oracle.com/apex/livesql/file/content_BM1LJQ87M5CNIOKPOWPV6ZGR3.html)

Reply:
cool. thanks for explaining.